.github/s2n_freebsd.sh

```diff
@@ -15,7 +15,12 @@
 set -eu
 export CTEST_PARALLEL_LEVEL=$(sysctl hw.ncpu | awk '{print $2}')
 
-cmake . -Bbuild -GNinja -DCMAKE_BUILD_TYPE=Debug
+cmake . -Bbuild -GNinja -DCMAKE_BUILD_TYPE=Release
 cmake --build ./build -j $CTEST_PARALLEL_LEVEL
 ninja -C build test
 cmake --build ./build --target clean #Saves on copy back rsync time
+
+cmake . -Bbuild -GNinja -DCMAKE_BUILD_TYPE=Debug
+cmake --build ./build -j $CTEST_PARALLEL_LEVEL
+ninja -C build test
+cmake --build ./build --target clean #Saves on copy back rsync time
```

Comment:
I think this will overwrite the build/Testing/Temporary directory with the results of the Debug run. If a test fails differently in release than it does for debug, we won't be able to see it in the test output artifact. Is this okay? An alternative could build in another directory and add another upload-artifact action to ci_freebsd.yml. But maybe this isn't a concern!